pkg/api/v1/clients/kubesecret/resource_client.go

```diff
@@ -81,19 +80,70 @@ func (rc *ResourceClient) toKubeSecret(resource resources.Resource) (*v1.Secret,
 	}, nil
 }
 
+func (rc *ResourceClient) fromPlainKubeSecret(secret *v1.Secret) (resources.Resource, error) {
+	resource := rc.NewResource()
+	// not our secret
+	// should be an error on a Read, ignored on a list
+	if len(secret.ObjectMeta.Annotations) == 0 || secret.ObjectMeta.Annotations[annotationKey] != rc.Kind() {
```

Comment:
Would it make sense to maybe log something here? Just so that we are aware of when these unexpected cases occur.

Reply:
Ok, makes sense